.github/workflows/push.yml

```diff
@@ -134,3 +137,57 @@ jobs:
         env:
           TARGET_VERSION: ${{ needs.set_version.outputs.version }}
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  release-macos:
+    needs:
+      - set_version
+      - cross-compile
+      - tests
+    runs-on: macos-latest
+    if: startsWith(github.ref, 'refs/tags/') && needs.set_version.outputs.is_semantic_tag == 'true'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Download-Binaries
+        uses: actions/download-artifact@v2
+        with:
+          name: darwin-binaries
+          path: release/
+      - name: Prepare files
+        run: |
+          mkdir release/signed
+          mkdir release/darwin-amd64
+          cp release/*amd64/kbc release/darwin-amd64
+          mkdir release/darwin-arm64
+          cp release/*arm64/kbc release/darwin-arm64
+      - name: Import Code-Signing Certificates
+        uses: Apple-Actions/import-codesign-certs@v1
+        with:
+          # The certificates in a PKCS12 file encoded as a base64 string
+          p12-file-base64: ${{ secrets.APPLE_DEVELOPER_CERTIFICATE_P12_BASE64 }}
+          # The password used to import the PKCS12 file.
+          p12-password: ${{ secrets.APPLE_DEVELOPER_CERTIFICATE_PASSWORD }}
+      - name: Install gon via HomeBrew for code signing and app notarization
+        run: |
+          brew tap mitchellh/gon
+          brew install mitchellh/gon/gon
+          brew install tree
+          tree release
+      - name: Sign the mac binaries with Gon
+        env:
+          AC_USERNAME: ${{ secrets.AC_USERNAME }}
+          AC_PASSWORD: ${{ secrets.AC_PASSWORD }}
+        run: |
+          gon -log-level=debug -log-json ./gon-amd64.json
```

Comment:
Je potrebny JSON log? Neda sa tam z toho potom velmi nic vycitat v CI.

Reply:
zmenim log level na `info`

Reply:
vid https://github.com/mitchellh/gon#machine-readable-output pise `Note you must specify both -log-level and -log-json.` - dam to aspon na to info.